File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/ETertiaryFixedDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object ETertiaryFixedDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd37
            AppTheme.STANDARD -> R.color.cstd37
            AppTheme.BLUE -> R.color.cbd37
            AppTheme.GREEN -> R.color.cgd37
            AppTheme.RED -> R.color.crd37
            AppTheme.NOTHING -> R.color.cnod37
        }
    }
}

File: lib/layer2_core/dark/e_tertiary_fixed_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class ETertiaryFixedDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd37;
      case 'standard': return cstd37;
      case 'blue': return cbd37;
      case 'green': return cgd37;
      case 'red': return crd37;
      case 'nothing': return cnod37;
      default: return cstd37;
    }
  }
}
